imgaug/parameters.py

```diff
@@ -753,6 +753,74 @@ def __str__(self):
         return "Normal(loc=%s, scale=%s)" % (self.loc, self.scale)
 
 
+class TruncNormal(StochasticParameter):
```

Comment:
I think "TruncatedNormal" might be the better name. Not sure how many people are familiar with the scipy function, but many people are going to know the tensorflow one `truncated_normal` and are hence going to be more familiar with the non-abbreviated form. For anybody not knowing either of the two, the non-abbreviated form is also going to be more obvious.